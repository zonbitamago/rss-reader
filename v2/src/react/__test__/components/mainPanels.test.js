'user strict';
jest.dontMock('../../components/mainPanels.jsx');
import React from 'react';
import renderer from 'react-test-renderer';
import MainPanels from '../../components/mainPanels.jsx';

test('snapshot', function() {
  const tree = renderer.create(
    <MainPanels/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
