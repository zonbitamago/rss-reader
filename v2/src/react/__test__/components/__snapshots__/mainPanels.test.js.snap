// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 記事が0個 1`] = `
<section
  className="mainPanels"
>
  <div
    className="ui blue inverted pointing secondary head menu"
  >
    <a
      className="link item"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="home large icon"
      />
    </a>
    <div
      className="item"
      onClick={[Function]}
    >
      <h3
        className="ui inverted header"
      >
        Home
      </h3>
    </div>
  </div>
  <div
    className="itemPanel"
  />
</section>
`;

exports[`snapshot 記事が1個 1`] = `
<section
  className="mainPanels"
>
  <div
    className="ui blue inverted pointing secondary head menu"
  >
    <a
      className="link item"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="home large icon"
      />
    </a>
    <div
      className="item"
      onClick={[Function]}
    >
      <h3
        className="ui inverted header"
      >
        Home
      </h3>
    </div>
  </div>
  <div
    className="itemPanel"
  >
    <div
      className="ui feed"
    >
      <div
        className="event"
      >
        <div
          className="label icon"
        >
          <img
            src="http://www.google.com/s2/favicons?domain=xxx"
          />
        </div>
        <div
          className="content"
        >
          <div
            className="summary"
          >
            name1
            <div
              className="date"
            >
              a few seconds ago
            </div>
          </div>
          <div
            className="extra"
          >
            <a
              href="javascript:void(0);"
              onClick={[Function]}
            >
              title1
            </a>
          </div>
        </div>
      </div>
      <div
        className="ui divider"
      />
    </div>
  </div>
</section>
`;

exports[`snapshot 記事が2個 1`] = `
<section
  className="mainPanels"
>
  <div
    className="ui blue inverted pointing secondary head menu"
  >
    <a
      className="link item"
      onClick={[Function]}
    >
      <i
        aria-hidden="true"
        className="home large icon"
      />
    </a>
    <div
      className="item"
      onClick={[Function]}
    >
      <h3
        className="ui inverted header"
      >
        Home
      </h3>
    </div>
  </div>
  <div
    className="itemPanel"
  >
    <div
      className="ui feed"
    >
      <div
        className="event"
      >
        <div
          className="label icon"
        >
          <img
            src="http://www.google.com/s2/favicons?domain=xxx"
          />
        </div>
        <div
          className="content"
        >
          <div
            className="summary"
          >
            name1
            <div
              className="date"
            >
              a few seconds ago
            </div>
          </div>
          <div
            className="extra"
          >
            <a
              href="javascript:void(0);"
              onClick={[Function]}
            >
              title1
            </a>
          </div>
        </div>
      </div>
      <div
        className="ui divider"
      />
    </div>
    <div
      className="ui feed"
    >
      <div
        className="event"
      >
        <div
          className="label icon"
        >
          <img
            src="http://www.google.com/s2/favicons?domain=xxx"
          />
        </div>
        <div
          className="content"
        >
          <div
            className="summary"
          >
            name2
            <div
              className="date"
            >
              a few seconds ago
            </div>
          </div>
          <div
            className="extra"
          >
            <a
              href="javascript:void(0);"
              onClick={[Function]}
            >
              title2
            </a>
          </div>
        </div>
      </div>
      <div
        className="ui divider"
      />
    </div>
  </div>
</section>
`;
